sklearn/feature_selection/_univariate_selection.py

```diff
@@ -361,6 +362,9 @@ def fit(self, X, y):
     def _check_params(self, X, y):
         pass
 
+    def _more_tags(self):
+        return {'allow_nan': False}
```

Comment:
This should probably have a `FIXME` or similar to say this should depend on the score func